crates/holochain_cascade/src/lib.rs

```diff
@@ -539,11 +536,11 @@ impl CascadeImpl {
     pub async fn fetch_record(
         &self,
         hash: AnyDhtHash,
-        options: NetworkGetOptions,
+        _options: NetworkGetOptions,
```

Comment:
Should this be respected here? `fetch` sounds like it's a network operation already and if it was a cascade function to `get or fetch` then it should just be called `get_record`?